docs/getting_started/tutorial-hyperbolic-discounting.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<p>Please do not write the params to the disk. I started it in the tutorials, but prefer how I continued in <a href="https://respy.readthedocs.io/en/latest/getting_started/tutorial-initial-conditions.html" target="_blank">https://respy.readthedocs.io/en/latest/getting_started/tutorial-initial-conditions.html</a>. The params are a string which is read with pandas and you can hide the options by loading them from the example model with <code>_, options = rp.get_example_model("robinson_crusoe_basic", with_data=False)</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>